src/test/resources/config/application.yml

```diff
@@ -73,8 +82,9 @@ info:
 #
 # Full reference is available at: https://www.jhipster.tech/common-application-properties/
 # ===================================================================
-
-jhipster:
+server:
```

Comment:
why jhipster properties was replaced with server?

Reply:
ServerProperties is used instead of JHipsterProperties

Reply:
fixed